hackathon banner

blackgirlbytes opened the pull request:
Added a temporary banner to promote our hackathon
<img width="1728" height="856" alt="Screenshot 2025-11-12 at 7 15 36 PM" src="https://github.com/user-attachments/assets/13968fd4-1730-44a7-9a99-b1c7a05da53e" />

blackgirlbytes committed:
Update hackathon banner with clearer, more compelling messaging

- Changed from 'CodeTV × goose hackathon - No keyboards allowed!' to more concise messaging
- New banner: 'Join our hackathon to build next-gen interfaces with goose and win prizes. Deadline Nov 14.'
- Hyperlinked 'Deadline Nov 14' instead of 'Join the challenge'
- Removed emojis for cleaner, more professional look
- Improved clarity by separating the deadline into its own sentence
- More compelling by leading with action and mentioning prizes

blackgirlbytes commented on documentation/src/theme/Root.tsx:
```diff
@@ -0,0 +1,42 @@
+      <div
+        style={{
+          backgroundColor: 'var(--ifm-color-primary)',
+          color: '#000',
```
yeah yeah yeah

blackgirlbytes commented on documentation/src/theme/Root.tsx:
```diff
@@ -0,0 +1,42 @@
+            fontWeight: '700',
+          }}
+        >
+          Deadline Nov 14
```
no i want it to say deadline

dianed-square reviewed (approved):
I'm assuming the black banner in the preview docs is a staging issue

blackgirlbytes committed:
Fix hackathon banner visibility in light mode

The hackathon banner was invisible in light mode due to black text on black background.
Changed backgroundColor from var(--ifm-color-primary) to fixed teal color #25c2a0
to ensure visibility in both light and dark modes.
